<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoTipWidget</class>
 <widget class="QWidget" name="AutoTipWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoTipWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelIcon">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="pixmap">
      <pixmap resource="../inst.qrc">:/images/Icon_76.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ElidedLabel" name="labelText">
     <property name="text">
      <string notr="true">text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StyleToolButton" name="toolButtonClose">
     <property name="text">
      <string/>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ElidedLabel</class>
   <extends>QLabel</extends>
   <header location="global">elidedlabel.h</header>
  </customwidget>
  <customwidget>
   <class>StyleToolButton</class>
   <extends>QToolButton</extends>
   <header location="global">styletoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../inst.qrc"/>
 </resources>
 <connections/>
</ui>
